data/phi1017/phi015/__cts__.xml

```diff
@@ -1,7 +1,8 @@
-<cts:work xmlns:cts="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1017.phi015" groupUrn="urn:cts:latinLit:phi1017" xml:lang="lat">
-  <cts:title xml:lang="lat">Epistulae</cts:title>
-  <cts:edition urn="urn:cts:latinLit:phi1017.phi015.perseus-lat2" workUrn="urn:cts:latinLit:phi1017.phi015">
-    <cts:label xml:lang="eng">Epistulae, Ad Lucilium epistulae morales Volume I-III</cts:label>
-    <cts:description xml:lang="eng">Seneca, Lucius Annaeus, ca. 4 B.C.-65 A.D, creator; Gummere, Richard M. (Richard Mott), 1883-, editor</cts:description>
-  </cts:edition>
-</cts:work>
\ No newline at end of file
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1017.phi015" groupUrn="urn:cts:latinLit:phi1017" xml:lang="lat">
+  <ti:title xml:lang="lat">Epistulae</ti:title>
+  <ti:edition urn="urn:cts:latinLit:phi1017.phi015.perseus-lat2" workUrn="urn:cts:latinLit:phi1017.phi015">
+    <ti:label xml:lang="lat">Epistulae</ti:label>
+    <ti:description xml:lang="eng">Seneca, Lucius Annaeus.  Ad Lucilium Epistulae Morales, Vol 1-3. Gummere, Richard M. (Richard Mott), editor.  
```

Comment:
Richard M. (Richard Mott)

Reply:
hi @lcerrato, sorry but I have the same question here, what is it you want me to change, since I see the full name spelled out in the file?